corehq/apps/hqwebapp/models.py

```diff
@@ -35,25 +35,29 @@ class MaintenanceAlert(models.Model):
 
     text = models.TextField()
     domains = ArrayField(models.CharField(max_length=126), null=True)
+    created_by_domain = models.CharField(max_length=255, null=True, db_index=True)
+    created_by_user = models.CharField(max_length=128, null=True)
 
     class Meta(object):
         app_label = 'hqwebapp'
+        db_table = 'hqwebapp_maintenancealert'
 
     @property
     def html(self):
         return mark_up_urls(self.text)
 
     def __repr__(self):
-        return "MaintenanceAlert(text='{}', active='{}', domains='{}')".format(
+        return "CommCareHQAlert(text='{}', active='{}', domains='{}')".format(
             self.text, self.active, ", ".join(self.domains) if self.domains else "All Domains")
```

Comment:
nit
```suggestion
        domains = ", ".join(self.domains) if self.domains else "All Domains"
        return f"{type(self).__name__}(text={self.text!r}, active={self.active!r}, domains={domains})"
```
Note intentional omission of `!r` on `domains` (and also no enclosing quotes) since the value of that attribute does not match what is printed in this representation, and what is printed is not a proper Python data structure that could be passed in to reconstruct this object.

Reply:
So instead of 
`CommCareHQAlert(text='Maintenance alert', active='False', domains='All Domains'`
this would now be
`CommCareHQAlert(text='Maintenance alert', active=False, domains=All Domains)`
or
`CommCareHQAlert(text='Maintenance alert', active=False, domains=test, test2)`

Confirming that this is what you expected how this would look like?

> what is printed is not a proper Python data structure that could be passed in to reconstruct this object.

Yup. For True/False this change seems okay, though seeing a comma separated list of domains without the enclosing quotes does look weird. And since we have used quotes for the text, it makes sense to use it for other string representations as well.
Or if we don't join domains, it would be
`CommCareHQAlert(text='Maintenance alert', active=False, domains=['test', 'test2'])`
which makes a bit more sense to me.

Reply:
> Confirming that this is what you expected how this would look like?

Yes.

> seeing a comma separated list of domains without the enclosing quotes does look weird

Maybe a little, but it communicates the value and it's obvious that that value cannot be used to reconstruct the object, unlike a properly formatted string, which would be misleading because it is not a valid value that can be used to reconstruct the object.

> if we don't join domains, it would be... `domains=['test', 'test2']`

That would be fine if you like that better. The down-side there is that an empty list means _all domains_ which may not be obvious.

Reply:
Thanks. 

If its okay, i am going to leave this as it is.
I don't see a valuable difference to what it is now and I prefer the quoted display for better grouping of text.

Reply:
What about "misleading because it is not a valid value that can be used to reconstruct the object"?

Reply:
I am not sure why that function will be used to reconstruct the object. I assume its only for display purpose?
And I believe it does not have any use really, i guess it was just added and can possibly be removed.
https://github.com/dimagi/commcare-hq/pull/17022/files

Reply:
Ah, good to know. I was not aware.

As I shared earlier
> And I believe it does not have any use really, i guess it was just added and can possibly be removed.
https://github.com/dimagi/commcare-hq/pull/17022/files

I went ahead and removed the function as it does not seem to really getting used anywhere except for tests which I then modified.
https://github.com/dimagi/commcare-hq/pull/33592/commits/61e1cfe13d132030859b2d63f976c2fbb565b4bf